actors/builtin/multisig/multisig_actor.go

```diff
@@ -23,15 +25,24 @@ func (t TxnID) Key() string {
 }
 
 type Transaction struct {
-	To     addr.Address
-	Value  abi.TokenAmount
-	Method abi.MethodNum
-	Params []byte
+        To         addr.Address
```

Comment:
theres a bunch of whitespace only changes here, did you run `go fmt` before pushing?

Reply:
Hi. The only commands I recall issuing are `make`, `make gen`, and `make test`. It's possible I ran something by accident and don't remember. It's also possible my IDE did something weird. Either way, if it's a problem, I can undo it one way or another, at least for the non-generated code.

Reply:
Yeah, your IDE might have decided to do some funny business...Running `go fmt` on the changed files should get you on the same formatting as us.

Reply:
Thanks. `gp fmt` did the trick.